### PR TITLE
Otel metrics doc fixes

### PR DESCRIPTION
**Pull request by turing85**
- Each sentence should be on a separate line
- Fix format error
- Fix typo

Separation into three commits is mainly for review. Feel free to squash on merge.

### CODE REVIEW

File: docs/src/main/asciidoc/opentelemetry-metrics.adoc [hunk 1]
```diff
@@ -322,20 +336,25 @@ counter.add(1, // <2>
...
+Using an unbounded set of dimensional data (many different values like a userId) can lead to a "cardinality explosion", an exponential increase in the creation of new time series.
+Avoid!
 
 OpenTelemetry provides many other types of Counters: `LongUpDownCounter`, `DoubleCounter`, `DoubleUpDownCounter` and also Observable, async counters like `ObservableLongCounter`, `ObservableDoubleCounter`, `ObservableLongUpDownCounter` and `ObservableDoubleUpDownCounter`.
 
 For more details please refer to the https://opentelemetry.io/docs/languages/java/instrumentation/#using-counters[OpenTelemetry Java documentation about Counters].
 
 === Gauges
-Observable Gauges should be used to measure non-additive values. A value that can increase or decrease over time, like the speedometer on a car. Gauges can be useful when monitoring the statistics for a cache or collection.
+Observable Gauges should be used to measure non-additive values.A value that can increase or decrease over time, like the speedometer on a car.
```
Comment (gsmet) [inline docs/src/main/asciidoc/opentelemetry-metrics.adoc:351]: You dropped a space here before A value.

**Review by gsmet — CHANGES_REQUESTED** [review]
I spotted a small issue. 
We haven’t enforced the one sentence per line rule (even if I agree it’s a lot better but YMMV). So while I have nothing against merging this as already done and also fixing some real issues, I would refrain to make additional similar PRs as it will make backporting doc changes that are usually no brainers prone to conflicts.

File: docs/src/main/asciidoc/opentelemetry-metrics.adoc (hunk 1 above)
Comment (turing85) [reply docs/src/main/asciidoc/opentelemetry-metrics.adoc]: Fixed.

**Comment by turing85** [comment]
> I spotted a small issue. We haven’t enforced the one sentence per line rule (even if I agree it’s a lot better but YMMV). So while I have nothing against merging this as already done and also fixing some real issues, I would refrain to make additional similar PRs as it will make backporting doc changes that are usually no brainers prone to conflicts.

The trigger for the change was not the format of the file, but an actual format error I notice. Not taken, I will refrain from reformatting the whole file.

**Review by gsmet — APPROVED** [review]

**Pull request merged**